pkgs/servers/keycloak/default.nix

```
@@ -7,7 +7,7 @@
 , callPackage
 , confFile ? null
 , plugins ? [ ]
-, extraFeatures ? [ ]
+, extraFeatures ? [ "hostname:v1" ]
```

Looks like this needs to be 
```suggestion
, extraFeatures ? [ "hostname-v1" ]
```

---

This fails even earlier, at the build time.

---

It's failing [here](https://github.com/keycloak/keycloak/blob/25.0.1/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/hostname/DefaultHostnameProvider.java#L310), might need to set `hostname-port`?

---

Doesn't work either.